logic/src/commonMain/kotlin/com/wire/kalium/logic/configuration/ServerConfig.kt

```diff
@@ -47,10 +48,10 @@ interface ServerConfigMapper {
 class ServerConfigMapperImpl : ServerConfigMapper {
     // TODO: url validation check e.g. remove https:// since ktor will control the http protocol
     override fun toDTO(serverConfig: ServerConfig): ServerConfigDTO =
-        with(serverConfig) { ServerConfigDTO(apiBaseUrl, accountsBaseUrl, webSocketBaseUrl, blackListUrl, teamsUrl, websiteUrl, title) }
+        with(serverConfig) { ServerConfigDTO(Url(apiBaseUrl), Url(accountsBaseUrl), Url(webSocketBaseUrl), Url(blackListUrl), Url(teamsUrl), Url(websiteUrl), title) }
 
     override fun fromDTO(serverConfigDTO: ServerConfigDTO): ServerConfig =
-        with(serverConfigDTO) { ServerConfig(apiBaseUrl, accountsBaseUrl, webSocketBaseUrl, blackListUrl, teamsUrl, websiteUrl, title) }
+        with(serverConfigDTO) { ServerConfig(apiBaseUrl.toString(), accountsBaseUrl.toString(), webSocketBaseUrl.toString(), blackListUrl.toString(), teamsUrl.toString(), websiteUrl.toString(), title) }
```

Comment:
I see many url to String and String to Url casting, do you think we can keep only the url and get rid of all the toString stuff?

Reply:
the `Url` class is ktor class and using it in each module means we need to include ktor in all the modules and make it something we depends on outside network
Kotlin does have a [URL](https://kotlinlang.org/api/latest/jvm/stdlib/org.w3c.dom.url/-u-r-l/) that target only Js

Reply:
Ok! Thanks for the clarification.